charts/plane-enterprise/templates/workloads/beat-worker.deployment.yaml

```
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: {{ .Release.Name }}-beat-worker
-        imagePullPolicy: Always
+        imagePullPolicy:  {{ .Values.services.api.pullPolicy | default "Always" }}
```

yamllint (1.35.1)

[warning] too many spaces after colon

(colons)